src/Symfony/Component/Config/Resource/ReflectionClassResource.php

```diff
@@ -149,6 +154,16 @@ private function generateSignature(\ReflectionClass $class)
                 yield print_r($defaults, true);
             }
         }
+
+        if ($class->isSubclassOf(EventSubscriberInterface::class)) {
```

Comment:
I'm wondering whether this interface coming from another component should really be hardcoded here. This means that any other package providing a similar feature has *no* way to invalidate the container.

Could we make a reusable implementation (using a different resource probably), which would work for such cases ?

Note that I have such use case in my own project, and I will soon have it in KnpMenuBundle too.

Reply:
@stof that's possible, but that's a new feature. Not the target here, but could be done in another PR.

Reply:
OK, actually, other packages can invalidate the cache, but suffer from the issue of invalidating them too often too.

Can you work on creating such new feature for 4.1 though ?

Reply:
> Can you work on creating such new feature for 4.1 though ?

not quite now, please open an issue :)

Reply:
done: https://github.com/symfony/symfony/issues/25984